app/content/profile/index.js

```diff
@@ -232,14 +211,11 @@ function Index() {
       <View style={styles.profileContainer}>
         <ProfileCard user={userData} email={userEmail} />
       </View>
-      <View>
-        <Text style={styles.heading}>Drill History</Text>
-      </View>
     </>
   );
 
   return (
-    <PaperProvider>
+    <PaperWrapper>
```

Comment:
What's the PaperWrapper vs. PaperProvider?

Reply:
PaperWrapper has the theme, and more importantly, the StatusBar component. StatusBar is to resolve the statusbar having white text on Android when in dark mode. It actually works when only placed only on the Assignment screen because it's a toggle, but I'm uncomfortable not having it everywhere, so PaperWrapper it is.

Reply:
It's a custom component that you can take a look